services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/LinkPropertiesSpec.groovy

```diff
@@ -212,7 +224,11 @@ class LinkPropertiesSpec extends BaseSpecification {
         islUtils.getIslInfo(links, isl.reversed).get().maxBandwidth == maxBandwidthValue.toInteger()
 
         and: "Delete link props"
-        northbound.deleteLinkProps(linkProps)
+        //DELETE /link/props deletes props for both directions, even if only one direction specified
+        def allLinkProps = northbound.getAllLinkProps()
```

Comment:
You don't need to get link props. Use variable `linkProps`. This variable contains one-link direction.

Reply:
thx